Jobsway2goMvc/Views/Jobs/Edit.cshtml

```diff
@@ -4,66 +4,118 @@
+                    </div>
+                </div>
+
+                <div class="col-md-6">
+                    <div class="form-group">
+                        <label for="maxSalary" class="form-label">Maximum Salary</label>
+                        <input asp-for="MaxSalary" type="number" class="form-control" id="maxSalary" placeholder="e.g. $30,000">
+                    </div>
+                </div>
+
+                <div class="col-md-6">
+                    <div class="form-group">
+                        <label for="location" class="form-label">Location</label>
+                        <input asp-for="Location" type="text" class="form-control" id="location" placeholder="e.g. London">
+                    </div>
+                </div>
+
+                <div class="col-md-12">
+                    <div class="form-group">
+                        <label for="requirements" class="form-label">Requirements</label>
+                        <textarea asp-for="Requirements" class="form-control requirements-area" id="requirements" rows="6" placeholder="Job Requirements"></textarea>
+                    </div>
+                </div>
+
+                <div class="col-md-12">
+                    <div class="form-group">
+                        <label for="description" class="form-label">Job Description</label>
+                        <textarea asp-for="Description" class="form-control description-area" id="description" rows="6" placeholder="Job Description"></textarea>
+                    </div>
+                </div>
+
+                <div class="col-md-12 text-center">
+                    <button type="submit" class="btn btn-primary post-btn">
+                        Edit Job
+                    </button>
+                    <a class="btn back-btn" asp-action="Index">Back to List</a>
+                </div>
             </div>
         </form>
     </div>
 </div>
 
-<div>
-    <a asp-action="Index">Back to List</a>
-</div>
\ No newline at end of file
+@section Scripts{
+    <script>
+        tinymce.init({
+            selector: 'textarea#requirements',
+            menubar: true,
+            plugins: [
+                'advlist', 'autolink', 'link', 'image', 'lists', 'charmap', 'preview',
+                'anchor', 'pagebreak',
+                'searchreplace', 'wordcount', 'visualblocks', 'visualchars', 'code',
+                'fullscreen', 'insertdatetime',
+                'media', 'table', 'emoticons', 'template', 'help'
+            ],
+            toolbar: 'undo redo | styles | bold italic | alignleft aligncenter alignright alignjustify | ' +
+                'bullist numlist outdent indent | link image | print preview media fullscreen | ' +
+                'forecolor backcolor emoticons | help',
+            advlist_bullet_styles: 'square',
+            advlist_number_styles: 'lower-alpha,lower-roman,upper-alpha,upper-roman'
+        });
+    </script>
```

Comment:
![edit job wysiwyg](https://user-images.githubusercontent.com/108676090/218776755-750f7c37-1178-40c9-ad47-75e5d50f1616.jpg)
data is now shown as intended after editing with wysiwyg